phpstan.neon

```diff
@@ -577,3 +577,7 @@ parameters:
 #            paths:
                 # weird case
 #                - packages/better-php-doc-parser/src/PhpDocParser/AnnotationContentResolver.php
+
+        -
+            message: '#Class cognitive complexity is 32, keep it under 30#'
+            path: rules/type-declaration/src/Rector/FunctionLike/ReturnTypeDeclarationRector.php
```

Comment:
Crossing cognitive complexity treshold should not be ignored.
Instead, should be solved by a new service extraction.

It works well with duplicated method PHPStan rule. That way we keep small consistent services with specific work and without duplication.

Reply:
fixed.